README.md

```diff
@@ -1,120 +1,174 @@
 
-```csharp
-string imageURL = imagekit.Url(new Transformation().Width(400).Height(300))
-    .Path("/default-image.jpg")
-    .UrlEndpoint("https://ik.imagekit.io/your_imagekit_id/endpoint")
-    .TransformationPosition("query")
-    .Generate();
+```cs
+string path = "/default-image.jpg";
+Transformation trans = new Transformation()
+                .Width(400)
+                .Height(300)
+                .AspectRatio("4-3")
+                .Quality(40)
+                .Crop("force").CropMode("extract").
+                Focus("left").
+                Format("jpeg").
+                Background("A94D34").
+                Border("5-A94D34").
+                Rotation(90).
+                Blur(10).
+                Named("some_name").
+                OverlayX(35).
+                OverlayY(35).
+                OverlayFocus("bottom").
+                OverlayHeight(20).
+                OverlayHeight(20).
+                OverlayImage("/folder/file.jpg"). // leading slash case
+                OverlayImageTrim(false).
+                OverlayImageAspectRatio("4:3").
+                OverlayImageBackground("0F0F0F").
+                OverlayImageBorder("10_0F0F0F").
+                OverlayImageDpr(2).
+                OverlayImageQuality(50).
+                OverlayImageCropping("force").
+                OverlayText("two words").
+                OverlayTextFontSize(20).
+                OverlayTextFontFamily("Open Sans").
+                OverlayTextColor("00FFFF").
+                OverlayTextTransparency(5).
+                OverlayTextTypography("b").
+                OverlayBackground("00AAFF55").
+                OverlayTextEncoded("b3ZlcmxheSBtYWRlIGVhc3k%3D").
+                OverlayTextWidth(50).
+                OverlayTextBackground("00AAFF55").
+                OverlayTextPadding(40).
+                OverlayTextInnerAlignment("left").
+                OverlayRadius(10).
+                Progressive(true).
+                Lossless(true).
+                Trim(5).
+                Metadata(true).
+                ColorProfile(true).
+                DefaultImage("folder/file.jpg/"). //trailing slash case
+                Dpr(3).
+                EffectSharpen(10).
+                EffectUsm("2-2-0.8-0.024").
+                EffectContrast(true).
+                EffectGray().
+                Original().
+                RawTransformation("h-200).w-300).l-image).i-logo.png).l-end");
+                
+  string imageURL = imagekit.Url(trans).Path(path).TransformationPosition("query").Generate();                
 ```
 
 This results in a URL like
 
```

Comment:
The above code won't produce the URL that is mentioned in the docs i.e.
```
https://ik.imagekit.io/your_imagekit_id/endpoint/tr:h-300,w-400/default-image.jpg
```

I would suggest give a simple example as we have mentioned all the methods in the table.